test/spec/map.js

```diff
@@ -53,11 +53,25 @@ describe('L.mapbox.map', function() {
                 done();
             });
 
-            server.respondWith("GET", "https://a.tiles.mapbox.com/v4/mapbox.map-0l53fhk2.json?access_token=key",
+            server.respondWith("GET", "https://api.mapbox.com/v4/mapbox.map-0l53fhk2.json?access_token=key&secure",
                 [200, { "Content-Type": "application/json" }, JSON.stringify(helpers.tileJSON)]);
             server.respond();
         });
 
+        it('loads styleJSON from a default mapbox ID', function(done) {
```

Comment:
Can we add a test like this for more GL styles?

Reply:
I hesitated to this, since there are 7 template styles and the functionality to intercept them & redirect to the styleJSON endpoint is the same for each:

https://github.com/mapbox/mapbox.js/blob/f9990352a3b4edeca4edca6e7cd035ffa4fd60c2/src/config.js#L8-L17

If you don't feel like it would clutter the test suite with too many redundant tests, then happy to go ahead and add 6 more. 👍

Reply:
Gotcha. Then I think just one test is fine.